react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -76,12 +84,12 @@ class Thumbnail extends Component {
         //    silence (& not even comfort noise).
         // 2. The audio is local. If we were to render local audio, the local
         //    participants would be hearing themselves.
-        const audioMuted = !audioTrack || audioTrack.muted;
-        const renderAudio = !audioMuted && !audioTrack.local;
+        const audioMuted = !_audioTrack || _audioTrack.muted;
+        const renderAudio = !audioMuted && !_audioTrack.local;
         const participantId = participant.id;
         const participantInLargeVideo
```

Comment:
Side though: Is this one of those cases where too much is being passed in? It seems like _largeVideo doesn't need to be passed in.

Reply:
As far as I know ```_largeVideo``` is used to decide on tinting